docs/changes.md

```diff
@@ -1,5 +1,9 @@
 #Changelog
 
+## 3.x.x
```

Comment:
you may set it to 3.3.0, though considering that we cannot release atm due to #107 I will not yet draft new releases unless bug fixes are made on master.